OPHD/States/MapViewState.h

```diff
@@ -249,6 +249,15 @@ class MapViewState : public Wrapper
 	Button mBtnToggleHeightmap;
 	Button mBtnToggleConnectedness;
 
+	// Bare Control's use for ToolTips
+	Control mTooltipSystemButton;
+	Control mTooltipCurrentTurns;
+	Control mTooltipPopulation;
+	Control mTooltipEnergy;
+	Control mTooltipFoodStorage;
+	Control mTooltipResourceStorage;
+	Control mTooltipResourceBreakdown;
```

Comment:
I feel like these should belong to corresponding Panel classes, rather than all lumped together here.

Reply:
These are 'hidden' or 'dummy' controls. Panels have no place in this context.

There is also no such beast in the current GUI implementation.

Reply:
Yeah, I think you're right. Sounds like we need an actual Control for these sorts of things.

Reply:
See #781 for thoughts on an implementation.